packages/components/package.json

```diff
@@ -23,10 +23,14 @@
     "./utilities/*": "./dist/utilities/*",
     "./translations/*": "./dist/translations/*"
   },
-  "files": ["dist"],
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "build": "node scripts/build.js",
-    "test": "pnpm build && web-test-runner --group default",
+    "test.setup": "mkdir -p dist/themes && pnpm run -C ../tokens build && cp ../tokens/dist/themes/light.css dist/themes/light.css",
+    "test.download": "pnpm playwright install-deps && pnpm playwright install",
```

Comment:
This is something one doesn't in day-to-day work. It looks like this is espeically for CI/CD and would therefore suggest to use tha playwright images as done in Solid: https://github.com/solid-design-system/solid/blob/c45e606ae9f6e8ada422e773dba24b4bf4101224/.github/workflows/check-pr.yml#L10

Reply:
This was altered to align it better. However, during testing, I ran into an issue described at https://github.com/microsoft/playwright/issues/6500. After adding the fix, everything worked. However, I don´t know why this does not seem to be an issue in Solid. Would recommend to have an eye on that if you will run into this issue there.